mods/ssb/moves.js

```diff
@@ -1298,18 +1302,17 @@ let BattleMovedex = {
 	// False
 	frck: {
 		accuracy: true,
-		basePower: 1000,
+		basePower: 0,
```

Comment:
```suggestion
		basePower: 1000,
```

Reply:
Is more fun to leave as is (for /dt ect)

Reply:
Ugh I keep hitting resolve 

Then this throws an error if it koes the user first and I don’t think there needs to be s target check

Reply:
Alright, leave as 0 then